src/silly_sprites/opengl/renderer.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <algorithm>
+#include <glm/glm.hpp>
+#include <memory>
+#include <vector>
+
+namespace sly::gl {
+
+    struct Command;
+    class BufferObject;
+    class ShaderProgram;
+
+    class Renderer final {
+    private:
+        std::vector<Command> m_command_buffer;
+        std::unique_ptr<BufferObject> m_buffer_object;
+
+    public:
+        Renderer();
+        ~Renderer();
```

Comment:
should we delete the copy ctor and copy assign here too?
copy a renderer sems not be good to me.

Reply:
Good idea, yes. Done.